lib/NormalModule.js

```diff
@@ -11,8 +11,6 @@ var ReplaceSource = require("webpack-sources").ReplaceSource;
 var CachedSource = require("webpack-sources").CachedSource;
 var LineToLineMappedSource = require("webpack-sources").LineToLineMappedSource;
 var ModuleParseError = require("./ModuleParseError");
-var TemplateArgumentDependency = require("./dependencies/TemplateArgumentDependency");
```

Comment:
This change was because after merging master into this branch, lint threw because these are unused. I have this in another PR, but so this passes. I wanted to note this change anyways.